scripts/lib/install-node

```diff
@@ -33,7 +34,11 @@ if ! check_version 2>/dev/null; then
     rm -rf /srv/zulip-node
     mkdir -p /srv/zulip-node
     tar -xJf "$tarball" --no-same-owner --strip-components=1 -C /srv/zulip-node
-    ln -sf /srv/zulip-node/bin/{node,npm,npx} /usr/local/bin
-    rm -rf /usr/local/nvm
+    ln -sf /srv/zulip-node/bin/{corepack,node,npm,npx} /usr/local/bin
+    COREPACK_DEFAULT_TO_LATEST=0 /usr/local/bin/corepack enable
+
+    # Clean up after previous versions of this script
+    rm -rf /srv/zulip-yarn /usr/bin/yarn /usr/local/nvm
```

Comment:
Would it be safer to write this as "if /srv/zulip-yarn exists, remove these files, with `/srv/zulip-yarn` last?"

Since we're probably leaving this code in place for a while after it's no longer needed, that seems like it would protect us from messing with users' systems that might happen to have a `/usr/bin/yarn` for some other reason.

Reply:
A server that hadn’t upgraded since 0ba9114c223ef881546bd74e8a53f8cce401f450 (#19018) will have `/usr/bin/yarn` and `/usr/local/nvm` but not `/srv/zulip-yarn`.

Reply:
OK, I guess that was new in Zulip 5.0, so we've got a while before we can drop that hunk. I think in production, we could do this conditionally in `upgrade-zulip-stage-2`, and in `provision`, we could use the "check for /srv/zulip-yarn"` rule, and I'd feel a lot better. (I think the main concrete risk I have in mind is anyone who is using `provision` on a Linux development server and also uses `nvm` for other projects).

Reply:
Oh, so could we just check whether `/usr/local/nvm` exists? I feel like `/usr/bin/yarn` is a path that some other software could be using, though?

I guess https://packages.debian.org/buster/all/yarnpkg/filelist suggests that the Debian package doesn't use that path, so yeah maybe this is fine.

Reply:
No, because after #19816 but before now, `/usr/local/nvm` would not exist (again, we have already been deleting it) but `/usr/bin/yarn` would.

There’s an unrelated `/usr/bin/yarn` in Debian’s [cmdtest package](https://packages.debian.org/bullseye/cmdtest) (which is why they install the package manager as `yarnpkg`). I would be willing to feel slightly bad about breaking it if we had historically been making an effort to cooperate with the user’s existing system, but we haven’t.